src/world_interface/world_interface.h

```diff
@@ -12,17 +14,20 @@
 #include <unordered_set>
 
 using namespace kinematics;
-// forward declare cam::CameraParams instead of including it
-// we do this to avoid unnecessarily including OpenCV in all build targets
-namespace cam {
-class CameraParams;
-}
 
 /**
  * @namespace robot
  * @brief Collection of functions that allows interfacing with the hardware and the world.
  */
 namespace robot {
+namespace types {
+class CameraHandle {
+public:
+	CameraHandle(std::shared_ptr<cam::Camera> cam) : camera(cam) {}
+private:
+	std::shared_ptr<cam::Camera> camera;
+};
```

Comment:
This is a dumb class that is used for reference counting without letting client code have access to the actual camera object. Is this the right place for it?